lib/connection.js

```diff
@@ -112,15 +113,26 @@ class Connection extends EventEmitter {
   }
 
   // Send a handshake packet over the connection
-  //   appName - name of an application to connect to
+  //   app - string or object, application to connect to as 'name' or
+  //         'name@version' or { name, version }
```

Comment:
Same here, I don't really care. @metarhia/jstp-core somebody wants to leave the third option here?